src/styles/actor/character/_sidebar.scss

```diff
@@ -54,7 +54,7 @@ aside {
     .sidebar {
         margin-bottom: 2.625rem;
         overflow: hidden auto;
-        padding: 0 var(--space-2) var(--space-2) var(--space-8);
```

Comment:
You tested how it looks like if it has a scrollbar?

Reply:
Stable scrollbar gutter, my dude!

Reply:
More wondering if its hugging the scrollbar if it shows up